javascript/src/refactor.js

```diff
@@ -478,11 +476,39 @@ class RefactorEngine {
         return assignments;
     }
 
+    adjustFunctionParams(parameterList) {
```

Comment:
Can we have some comments here - what is `adjustFunctionParams` doing?

Reply:
Yes added necessary comments